internal/manager/config_validation.go

```diff
@@ -59,6 +59,9 @@ func (c *Config) validateKonnect() error {
 		return nil
 	}
 
+	if c.KongAdminSvc.Name == "" && c.KongAdminSvc.Namespace == "" {
+		return errors.New("--kong-admin-svc has to be set when using --konnect-sync-enabled")
```

Comment:
Have we considered edge cases where `--kong-admin-svc` = `/...` or `.../` ? In such cases, one of `Name` and `Namespace` will be empty.

Reply:
It's already validated by the `namespacedNameFromFlagValue` when building the value of this variable, but yeah, it won't hurt us to change the `&&` to `||` 👍 : https://github.com/Kong/kubernetes-ingress-controller/pull/3598/commits/4c6650b3e6a8ddf4994f5064eb73a43f1afc6067